.circleci/config.yml

```diff
@@ -132,6 +133,10 @@ jobs:
           name: Make bootstrap
           command: make bootstrap
 
+      - run:
+          name: Pre-load simulator
+          command: *preload_simulator
```

Comment:
When we had this in place before we sometimes followed it with a `sleep 15` command so that it would wait a while before running the next step. I'd like to try it without that first so see if this solves it on its own.